skylog/storage/cloudspanner/schema.sdl

```diff
@@ -0,0 +1,9 @@
+-- TreeNodes stores Merkle tree nodes. The nodes are sharded according to
+-- ShardId, which is computed based on the tree shape and write pattern, in
+-- order to maximize writing parallelism.
+CREATE TABLE TreeNodes (
+  TreeId       INT64 NOT NULL,
+  ShardId      INT32 NOT NULL,
+  NodeId       UINT64 NOT NULL,
+  SubtreeHash  BYTES(32) NOT NULL,
```

Comment:
I think this is a hash of the nodes as stored and is not related to tree hashing? If so then that might be a useful comment to add.